src/commands/AdventofcodeCommand.ts

```diff
@@ -0,0 +1,81 @@
+import { Message, MessageEmbed } from "discord.js";
+import Command from "../abstracts/Command";
+import AdventOfCodeService from "../services/AdventOfCodeService";
+import { EMBED_COLOURS, ADVENTOFCODE_LEADERBOARD, ADVENTOFCODE_YEAR, ADVENTOFCODE_INVITE } from "../config.json";
```

Comment:
I think the config variables should consider "Advent of Code" as three different words, so we would name them `ADVENT_OF_CODE_LEADERBOARD`, `ADVENT_OF_CODE_YEAR`, `ADVENT_OF_CODE_INVITE` and so on. I think it's more consistent and makes it more readable. Just an opinion tho